docs/registry-public.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: docs
+title: "Using a public registry"
+permalink: /docs/registry-public
+---
+
+# Working with a public registry
+
+After building an image with `docker build . -t mynginx:local`, it can be
```

Comment:
the "with `docker build . -t mynginx:local`" refers to the intro we used to have in this doc. Please remove it.

The first paragraph reads better like so:
"After building an image we can push it to one of the mainstream public registries. For this example we created an account with [https://hub.docker.com/]()  and we log in as `kjackal`."

Reply:
done